drivers/Makefile.dep

```diff
@@ -32,6 +32,21 @@ ifneq (,$(filter cc110x,$(USEMODULE)))
   endif
 endif
 
+ifneq (,$(filter cc2420,$(USEMODULE)))
+  USEMODULE += cc2420
```

Comment:
Yeah once I analysed I saw that, thanks for the explanation. Now `USEMODULE += cc2420` it's removed.